CanDIG/htsget_app
# Misc fixes

I don't have jira tickets for these, but while working on DIG-910, I found some other problems in htsget:

* multiple region search now works
* better error handling for when htsget can't find a file for a drs_object
* the data endpoints were performing incorrectly:
  - the variants/{id} endpoint should return an array of variants/data/{id} URLs that, concatenated together, constitute a VCF file. Minimally, header + body should be returned as separate URLs.
  - search now returns a property "htsget" which contains the same sort of HTSGetURL object specified in openapi.yml.

Pytest should still work correctly and you should be able to see the new formats in the Paw request for variant search:
* from CanDIGv2, python settings.py, source env.sh, export TESTENV_URL=http://docker.localhost:3333
* cd lib/htsget-server
* pip install -r requirements.txt
* cd htsget_app
* pytest
* go to Paw and run the "add items to dataset" request to add the test objects to a dataset user1 can see
* run the variant search request
